quickstack/files/sahara_policy.json

```diff
@@ -0,0 +1,72 @@
+{
+    "default": "",
+
+    "data-processing:clusters:get_all": "role:sahara_user",
+    "data-processing:clusters:create": "role:sahara_user",
+    "data-processing:clusters:scale": "role:sahara_user",
+    "data-processing:clusters:get": "role:sahara_user",
+    "data-processing:clusters:delete": "role:sahara_user",
+    "data-processing:clusters:modify": "role:sahara_user",
+
+    "data-processing:cluster-templates:get_all": "role:sahara_user",
+    "data-processing:cluster-templates:create": "role:sahara_user",
+    "data-processing:cluster-templates:get": "role:sahara_user",
+    "data-processing:cluster-templates:modify": "role:sahara_user",
+    "data-processing:cluster-templates:delete": "role:sahara_user",
+
+    "data-processing:node-group-templates:get_all": "role:sahara_user",
+    "data-processing:node-group-templates:create": "role:sahara_user",
+    "data-processing:node-group-templates:get": "role:sahara_user",
+    "data-processing:node-group-templates:modify": "role:sahara_user",
+    "data-processing:node-group-templates:delete": "role:sahara_user",
+
+    "data-processing:plugins:get_all": "role:sahara_user",
+    "data-processing:plugins:get": "role:sahara_user",
+    "data-processing:plugins:get_version": "role:sahara_user",
+    "data-processing:plugins:convert_config": "role:sahara_user",
+
+    "data-processing:images:get_all": "role:sahara_admin",
+    "data-processing:images:get": "role:sahara_admin",
+    "data-processing:images:register": "role:sahara_admin",
+    "data-processing:images:unregister": "role:sahara_admin",
+    "data-processing:images:add_tags": "role:sahara_admin",
+    "data-processing:images:remove_tags": "role:sahara_admin",
+
```

Comment:
There should be two roles: sahara_user and sahara_admin. sahara_admin will have the additional permission of managing the Sahara image registry. I do not want to give this role to all users since there is essentially one global image registry... not individualized to each project/tenant.